frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -238,7 +238,7 @@ export default Vue.extend({
     },
     methods: {
         format(value: BigNumber): string {
-            return formatToAutomaticDecimalPoints(value);
+            return formatToAutomaticDecimalPoints(value, undefined, { formatWithThousandSeparators: true });
```

Comment:
If it's `true` by default, you don't need to change this call, right?

Reply:
> you don't need to change this call

This wasn't addressed, right? (there is even no `formatWithThousandSeparators` anymore)

I also wonder why wasn't catch by the types?

Reply:
Hmm, I am wondering that as well. The tests should have failed IMO. I fixed it now!